kedro/templates/project/prompts.yml

```diff
@@ -1,40 +1,43 @@
+project_name:
+  title: "Project Name"
+  text: |
+    Please enter a human readable name for your new project.
+    Spaces, hyphens, and underscores are allowed.
+    To skip this step in future use --name
+  regex_validator: "^[\\w -]{2,}$"
+  error_message: |
+    It must contain only alphanumeric symbols, spaces, underscores and hyphens and
+    be at least 2 characters long.
+
 add_ons:
   title: "Project Add-Ons"
   text: |
-    Here you can select which add-ons you'd like to include. By default, none are included.
-    To read more about these add-ons and what they do visit: kedro.org/{insert-documentation}
+    These optional tools are here to help you apply software engineering best practices to your project quickly and easily.
+    To skip this step in future use --add-ons
+    To read more about these add-ons and what they do visit: kedro.org/ # TODO - add docs link in #3048
 
     Add-Ons
-    1) Linting : Provides a basic linting set up with Black and ruff
-    2) Testing : Provides basic testing set up with pytest
-    3) Custom Logging : Provides more logging options
-    4) Documentation: Provides basic documentations setup with Sphinx
-    5) Data Structure: Provides a directory structure for storing data
-    6) Pyspark: Provides set up configuration for working with PySpark
-    7) Kedro Viz: Provides Kedro's native visualisation tool
+    1) Lint: Provides a basic linting set up with Black, Ruff
+    2) Test: Provides basic testing set up with pytest
```

Comment:
```suggestion
    2) Test: Basic testing set up with pytest
```

Reply:
Please can you remove the "Provides" in each case...? On second thoughts, maybe I'll make a commit on your branch?

Reply:
No worries, it's been dropped in [c5c828e](https://github.com/kedro-org/kedro/pull/3355/commits/c5c828e52daa5cb82d38510cfc44875fd62da06d)